code/modules/surgery/organs/augments_arms.dm

```diff
@@ -20,7 +22,16 @@
 
 	update_icon()
 	SetSlotFromZone()
-	items_list = contents.Copy()
+	create_items(normal_items)
+
+/obj/item/organ/cyberimp/arm/proc/create_items(var/list/arg_items, var/emag = 0)
+	for(var/listitem in arg_items)
+		if(ispath(listitem) && !(locate(listitem) in items_list))
```

Comment:
Just let it error if something isn't a path. In the same vein let it add a second of the same item.